### ISSUE TITLE
Awning "slider" UI not correct

### ISSUE THREAD

**Issue by bitcrumb** [home-assistant/frontend#18254]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The slider UI used to show the opening/closing progress of a cover is illogical when used for an awning. I would expect the slider UI to be inverted. With which I mean that the "negative" white space in the screen below should actually be purple.

In other words, since the awning is only opened for 19% the purple color/handle should only equal 19% of the height of the slider. While currently it effectively is 81% of the slider.

This makes sense for "screens" (which are also of the type "cover"). But not for awning (for which the open/close is inverted too).

TLDR: Screenshot below is my awning being open at 19% (this number is correct), but the purple slider is actually 81% big. I would expect the purple slider handle to be 19% of height and increase with the opening of the awning increasing.

![Screenshot 2023-10-16 at 09 17 30](https://github.com/home-assistant/frontend/assets/135126/94f34a52-f08a-444b-ae92-aa4b8cbea129)

### Describe the behavior you expected

I would expect the UI to differentiatie between screens & awnings. Opening a screen is not the same as opening an awning, as demonstrated by the fact that "open" and "close" mean something else for both, even though they are both of the type "cover".

### Steps to reproduce the issue

No steps needed to reproduce, as this has been wrong since the UI was refreshed.

### What version of Home Assistant Core has the issue?

2023.10.3
